### PR TITLE
id3tool: init at 1.2a

### LINKED ISSUE #157008: id3tool
**Project description**
id3tool is a command line utility for easy manipulation of the ID3 tags present in MPEG Layer 3 audio files

**Metadata**

* homepage URL: http://nekohako.xware.cx/id3tool/
* source URL: http://nekohako.xware.cx/id3tool/id3tool-1.2a.tar.gz
* license: homegrown/free (See COPYING in source)
* platforms: i386/Linux,  x86/Win32

**Homebrew bottle**

For reference: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/id3tool.rb

### PR DESCRIPTION
**Pull request by danielb2**
###### Motivation for this change

Add derivation for https://github.com/NixOS/nixpkgs/issues/157008

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Closes: #157008

### CODE REVIEW

**Comment by jansol** [comment]
Consider adding yourself in `meta.maintainers` if you're planning to make sure the derivation stays intact.

**Comment by jansol** [comment]
Consider adding yourself in `meta.maintainers` if you're planning to make sure the derivation stays intact.

**Comment by danielb2** [comment]
> Consider adding yourself in `meta.maintainers` if you're planning to make sure the derivation stays intact.

what does this mean? In what way would it become damaged if I'm not in the maintainers?

**Comment by jansol** [comment]
It could for example stop building as a result of a compiler update, or the source URL could get retired.

**Comment by danielb2** [comment]
I'm OK to be off the list. Thanks.

**Comment by onny** [comment]
@ofborg build id3tool

File: pkgs/applications/audio/id3tool/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "id3tool";
+  version = "1.2a";
+
+  src = fetchurl {
+    url = "http://nekohako.xware.cx/id3tool/id3tool-1.2a.tar.gz";
+    sha256 = "eQjWbFqr4qU66AGegjT0IxSF2Avksv5yydBAE8/xyuw=";
```
Comment (drupol) [inline pkgs/applications/audio/id3tool/default.nix:9]:
```suggestion
    hash = "sha256-eQjWbFqr4qU66AGegjT0IxSF2Avksv5yydBAE8/xyuw=";
```

File: pkgs/applications/audio/id3tool/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl }:
```
Comment (drupol) [inline pkgs/applications/audio/id3tool/default.nix:1]:
```suggestion
{ lib, stdenvNoCC, fetchurl }:
```

File: pkgs/applications/audio/id3tool/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+stdenv.mkDerivation {
```
Comment (drupol) [inline pkgs/applications/audio/id3tool/default.nix:3]:
```suggestion
stdenvNoCC.mkDerivation {
```

**Review by drupol — CHANGES_REQUESTED** [review]
Hi !

I'm a new committer and I'm doing some kind of cleanup.

Would you be kind to make the necessary changes so we can move forward?

File: pkgs/applications/audio/id3tool/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    install -D id3tool "$out/bin/id3tool"
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "ID3 editing tool";
+    homepage = "http://nekohako.xware.cx/id3tool/";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [  ];
```
Comment (drupol) [inline pkgs/applications/audio/id3tool/default.nix:24]: Consider adding yourself as a maintainer.

**Review by drupol — CHANGES_REQUESTED** [review]

**Comment by danielb2** [comment]
I'm closing this because I keep hearing about adding myself as a maintainer, it's taken over a year, and it's not worth hearing about this anymore. Clearly no one else cares, or it would have been merged.

**Pull request closed without merging**